examples/ucp_client_server.c

```diff
@@ -1071,11 +1301,23 @@ int main(int argc, char **argv)
     char *server_addr = NULL;
     char *listen_addr = NULL;
     int ret;
+    char chunk_size_env[256];
+    char max_buffs_env[256];
 
     /* UCP objects */
     ucp_context_h ucp_context;
     ucp_worker_h  ucp_worker;
 
+    /*
+     * Modify environment to prevent RX hdr and payload mpools from growing. 
```

Comment:
explain why they should not grow

Reply:
could remove since it's default in IB TL